src/dbt_client/dbtCloudIntegration.ts

```diff
@@ -421,6 +422,10 @@ export class DBTCloudProjectIntegration
     );
   }
 
+  async generateDocsImmediately(command: DBTCommand) {
```

Comment:
This seems to be more generic method, should this still be tied to docs?

Reply:
Not sure. Other commands  are implemented in same way.

Reply:
Because the commands needs to be slightly modified based on cloud vs core. They are currently not typed.